client/viame-web-common/components/Viewer.vue

```diff
@@ -382,8 +383,12 @@ export default defineComponent({
 <template>
   <v-main class="viewer">
     <v-app-bar app>
-      <navigation-title />
       <slot name="title" />
+      <span
+        class="title pl-3"
+      >
+        {{ datasetName }}
+      </span>
```

Comment:
Likewise, move dataset name down to Viewer, because both platforms will have the same source of name (datasetmeta.name)